runtime/runtime/src/lib.rs

```diff
@@ -1589,8 +1589,7 @@ impl Runtime {
         let apply_state = &mut processing_state.apply_state;
         let state_update = &mut processing_state.state_update;
 
-        let signed_txs =
-            processing_state.transactions.transactions.into_iter().cloned().collect::<Vec<_>>();
+        let signed_txs = processing_state.transactions.par_iter_nonexpired_transactions().cloned();
```

Comment:
Have we added or will we add some tests where we ensure that we are not processing expired txs?

Reply:
Oh yeah I should restore some of those I had in that other PR attempt.